WebRoot/messages/ZmMsg.properties

```diff
@@ -666,6 +666,7 @@ clientLoginNotice = <a target="_new" href="https://www.zimbra.com">Zimbra</a> ::
     <a target="_new" href="https://blog.zimbra.com">Blog</a> - \
     <a target="_new" href="https://wiki.zimbra.com">Wiki</a> - \
     <a target="_new" href="https://www.zimbra.com/forums">Forums</a>
+clientUpdatedLoginNotice = © 2020 Synacor, Inc. All rights reserved. "Zimbra" is a registered trademark of Synacor, Inc.
```

Comment:
why we have added this trademark key extra? isn't this already part of existing keys?

Reply:
have done the changes for the year